SPECS/aspnetcore-runtime-3.1/aspnetcore-runtime-3.1.spec

```diff
@@ -35,7 +35,13 @@ cp -r shared/Microsoft.AspNetCore.App %{buildroot}%{_libdir}/dotnet/shared
 %{_libdir}/dotnet/shared/Microsoft.AspNetCore.App
 
 %changelog
+* Wed May 12 2021 Andrew Phelps <anphel@microosft.com> - 3.1.15-1
```

Comment:
I think this is are result of something going wrong during a merge: why are these spec changes part of this PR?